content/strategy-goals/strategy/code-intelligence/index.md

```diff
@@ -106,25 +105,28 @@ Building the code graph also means we need to generate and store increased amoun
 
 Our next step is helping customers set up, monitor and weed out any issues that might arise from our first trials to move auto-indexing into Beta. At this point we’ll be aiming to roll it out to a larger number of customers.
 
-**Make code navigation feel fast and reliable**
+**Unlock dependency navigation**
 
-We receive a significant amount of feedback around papercuts and potential improvements to our navigation experience. We are commited to making our navigation feel fast and reliable to users. Our current focus is in solving low hanging navigation issues while also adding quality of life improvements with the aim of increasing code navigation usage.
+We believe this is emerging as an additional killer feature. Most developers think in terms of package and dependency versions rather than focusing on external repositories. Supporting precise dependency navigation elevates the code navigation experience to a new level of cross-project analysis. Dependency navigation coupled with cross-repository takes the IDE model and scales it to encompass the whole cognitive space of a company.
```

Comment:
Do we want to include a more concrete use case?  Something around internal users being able to see if a function is able to be removed/deprecated?  For security folks they're able to see if a function is called anywhere (whether in their code or somewhere else in their deps graph?)